packages/api/src/controllers/stripe.ts

```diff
@@ -786,6 +803,13 @@ app.post("/migrate-test-products", async (req, res) => {
       stripeCustomerPaymentMethodId: null,
       newStripeProductId: null,
     });
+  } else {
+    res.json({
```

Comment:
Probably set a status code here too
```suggestion
    res.status(500).json({
```